CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Author names are now displayed in alphabetical order by last name, falls back on first name if necessary
 - Ability to output sharing links within an Image and Text 50/50 Group module
 - Google Optimize code on `find-a-housing-counselor` page
+- Added a test for get_browsefilterable_posts function of the sublanding page 
```

Comment:
@grapesmoker Already merged this in, but for future reference - I don't think you need to add tests to the `CHANGELOG`; in this case I asked you to update it because you fixed something, so that'd go under the "Fixed" section.